scenario-tests.js/index.js

```diff
@@ -0,0 +1,69 @@
+// const tape = require('tape')
+
+// these should be already set when the conductor starts it is the same for every test
+const TEST_INTERFACE_URL = 'ws://localhost:3000'
+const TEST_INTERFACE_ID = 'test-interface'
+
+// helpers for creating a config
+const Config = {
+  dna: (path, id = path) => ({ path, id }),
+  agent: (name, id = name) => ({ id, name }),
+  instance: (agent, dna, id = `${dna.id}::${agent.id}`) => ({
+    id,
+    agent_id: agent.id,
+    dna_id: dna.id
+  })
+}
+
+/**
+ * Represents a conductor process to which calls can be made via RPC
+ *
+ * @class      Conductor (name)
+ */
+class Conductor {
+  constructor (config, connect) {
+    this.config = config
+    this.webClientConnect = connect
+  }
+
+  async connect () {
+    const { call } = await this.webClientConnect(TEST_INTERFACE_URL)
+    this.call = call
+  }
+
+  /**
+   * Calls the conductor RPC functions to initialize it according to the config
+   */
+  async initialize () {
+    // setup the DNAs
+    // dna = {id, path}
+    const call = this.call
+    await this.config.dnas.forEach(async dna => {
```

Comment:
@willemolding if we have all the info at time of initialization, why not initialize based on a config file instead? how were you thinking the conductor would get started?